docs/wiki/research/roadmap.md

```diff
@@ -10,16 +10,16 @@ Even though there is no official roadmap and no authority which could dictate it
 
 ## Core R&D
 
-The discussion, resources and all research and development on the core protocol is fully open, free and public. Anyone can learn about it (as you are probably doing in this wiki) and further more, anyone can participate. There is no set of individuals which could push core protocol changes, the Ethereum community can raise the voice to help steer the discussion. To learn more about the core R&D shaping the protocol, read the [wiki page about it](/wiki/dev/core-development.md).
+The discussion, resources and all research and development on the core protocol is fully open, free and public. Anyone can learn about it (as you are probably doing in this wiki) and further more, anyone can participate. There is no set of individuals which could push core protocol changes, the Ethereum community can raise the voice to help steer the discussion. To learn more about the core R&D shaping the protocol, read the [wiki page about it](../dev/core-development.md).
```

Comment:
I think it's not a good idea to use relative paths for links. It's better to use path with root directory

Reply:
Thank you Kira, the reason was because 
index.html uses /main/docs + vm.route.file(https://github.com/eth-protocol-fellows/protocol-studies/blob/main/docs%2Findex.html#L333) to build the epf page, so the page gives 404 errors when accessing links and not rendering images. 

So if the epf.wiki url is built like that, how to fix it?

Reply:
thank you @shyam-patel-kira, I'll fix in the next commit.

- table cells foot-notes are not parsing either in html, so probably I need to use p or span wrap, any thoughts?

Reply:
Can either use html or use this markdown guide [link](https://www.markdownguide.org/extended-syntax/), it's super handy